src/test/java/htsjdk/tribble/annotation/StrandTest.java

```diff
@@ -0,0 +1,94 @@
+package htsjdk.tribble.annotation;
```

Comment:
these tests aren't running for some reason...there is no tests coverage in Strand.java

Reply:
Just submitted a commit extending HtsjdkTest. Hopefully that fixes it.

Reply:
That did the trick, thanks!